setup.py

```diff
@@ -77,6 +77,18 @@
         'plotly': ['plotly>=3.6.0'],
     },
     'include_package_data': True,
+
+    'classifiers': [
+        'Development Status :: 6 - Mature',
+        'Intended Audience :: Education',
+        'Intended Audience :: Science/Research',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```

Comment:
do we need to specify 3.6?

Reply:
Good question. I don't know. In this [example](https://packaging.python.org/guides/distributing-packages-using-setuptools/#classifiers) they seem to specify each version. Maybe it's best to stay vague...

Reply:
I think you need to specify all versions that are supported - in the example, they don't support 3.1, for instance....